plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/artifacts/ManifestFetcher.kt

```diff
@@ -41,6 +38,10 @@ class ManifestFetcher(
             .resolve("jetbrains-lsp-manifest.json")
     }
 
+    private val lspManifestUrl
+        get() = getManifestEndpoint()
```

Comment:
We removed this from companion object since this function should not be called during build time